app/src/main/res/layout/fragment_client_settings.xml

```diff
@@ -1,14 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout
+<androidx.constraintlayout.motion.widget.MotionLayout
         xmlns:android="http://schemas.android.com/apk/res/android"
         xmlns:app="http://schemas.android.com/apk/res-auto"
         xmlns:tools="http://schemas.android.com/tools"
+        android:id="@+id/settingsContainer"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:background="@color/primary_dark"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/c7380d74fc32b4ddceef02c8d9a561008c986f4f/app/src/main/res/layout/fragment_client_settings.xml#L9"></span>Possible overdraw: Root element paints background <code>@color/primary_dark</code> with a theme that also paints a background (inferred theme is <code>@style/AppTheme</code>)</td>
    </tr>
  </tbody>
</table>